src/com/raghav/javabasics/CheckNumberIsBinary.java

```diff
@@ -0,0 +1,28 @@
+package com.raghav.javabasics;
+// Write a java program to check whether given number is binary or not. A binary number is a number which contains only 0 or 1.
+
+public class CheckNumberIsBinary {
+
+    public static void main(String args[]) {
```

Comment:
For all the programs, can you add the run output in the description to identify what the output of each programs are?

Reply:
output: binary
not binary
not binary
binary
binary